src/flamegraph/mod.rs

```diff
@@ -339,7 +371,8 @@ where
         };
         filled_rectangle(&mut svg, &mut buffer, &rect, color, &mut cache_rect)?;
 
-        let fitchars = (rect.width() as f64 / (FONTSIZE as f64 * FONTWIDTH)).trunc() as usize;
+        let fitchars =
+            ((x2 - x1) as f64 / (opt.font_size as f64 * opt.font_width)).trunc() as usize;
```

Comment:
We should probably keep using `rect.width()` here.

Reply:
missed during initial rebase.